looprpc/client.proto

```diff
@@ -783,25 +783,25 @@ message LiquidityParameters {
     uint64 failure_backoff_sec = 9;
 
     /*
-    Set to true to enable automatic dispatch of loop out swaps. All swaps will
-    be limited to the fee categories set by these parameters, and total
-    expenditure will be limited to the auto out budget.
+    Set to true to enable automatic dispatch of swaps. All swaps will be limited 
+    to the fee categories set by these parameters, and total expenditure will 
+    be limited to the autoloop budget.
     */
-    bool auto_loop_out = 10;
+    bool autoloop = 10;
```

Comment:
This will possibly break REST users if they referenced the parameter by name. But I guess not many people are using it yet anyway and we declared it as experimental so this is OK?

Reply:
It is a breaking change, but since it's a rename only, I'd say it's less risky to do.

Reply:
> I guess not many people are using it yet anyway and we declared it as experimental so this is OK?

This was my reasoning, and I don't think we will break anybody's scripts/code so I think it's worth the rename. 
Can also deprecate and add new fields if we're uncomfortable with it. 

@alexbosworth WDYT?

Reply:
Discussion in loop notes, ok with breaking change for experimental/low use API.